### ISSUE TITLE
Accessibility: add missing labels to gallery thumbnail size controls

### ISSUE THREAD

**Issue by demiankatz** [UniversalViewer/universalviewer#1488]
The slider control to adjust thumbnail size in the gallery lacks labels. These should be added, in an appropriately i18n-friendly way.

See https://github.com/IIIF-Commons/iiif-gallery-component/pull/1 for work-in-progress -- this was not completed before the gallery component was reintegrated into the main UV application, so we should modernize and finish it now.
